src/web-page.ts

```diff
@@ -1,7 +1,7 @@
 export class WebPage {
 
     public protocol : string;
-    public headers : object;
+    public headers : any;
```

Comment:
That was weird. I could not iterate over all properties of `headers` if a for each loop as long as it remained `object` so I changed it to `any`. `object` is not a basic TypeScript type. I guess it has to be `Object`.